crates/router/src/connector/aci.rs

```diff
@@ -51,6 +51,14 @@ impl api::PaymentVoid for Aci {}
 impl api::PaymentCapture for Aci {}
 impl api::PaymentSession for Aci {}
 impl api::ConnectorAccessToken for Aci {}
+impl api::PaymentToken for Aci {}
+
+impl
```

Comment:
Add this implementation in the default_impl macro in flow.rs file
https://github.com/juspay/hyperswitch/blob/main/crates/router/src/core/payments/flows.rs#L116